src/browser/index.jsx

```diff
@@ -64,34 +70,39 @@ function showUnreadBadgeOSX(unreadCount, mentionCount) {
   }
 
   ipcRenderer.send('update-unread', {
+    sessionExpired,
     unreadCount,
     mentionCount,
   });
 }
 
-function showUnreadBadgeLinux(unreadCount, mentionCount) {
+function showBadgeLinux(sessionExpired, unreadCount, mentionCount) {
   if (remote.app.isUnityRunning()) {
-    remote.app.setBadgeCount(mentionCount);
+    if (sessionExpired) {
+      remote.app.setBadgeCount(1);
```

Comment:
Given the limitation here, I opted to interpret the specs to show a `1` instead of no badge at all.

Reply:
Wouldn't this override an existing actual count from other instances?

Reply:
Ah, good catch! Yes, it probably makes sense to increment `mentionCount` here instead.